packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -141,18 +147,65 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   const isInLobbyOrConnecting = currentPage === 'lobby';
   const hasJoinedCall = !!(currentPage && hasJoinedCallFn(currentPage, currentCallState ?? 'None'));
 
+  const callComposite = (
+    <CallComposite
+      {...props}
+      formFactor={formFactor}
+      options={{ callControls: false }}
+      adapter={callAdapter}
+      fluentTheme={fluentTheme}
+    />
+  );
+
+  const callWithChatControlBar = (
+    <CallWithChatControlBar
+      callAdapter={callAdapter}
+      chatAdapter={chatProps.adapter}
+      chatButtonChecked={showChat}
+      onChatButtonClicked={toggleChat}
+      peopleButtonChecked={showPeople}
+      onPeopleButtonClicked={togglePeople}
+      mobileView={isMobile}
+      disableButtonsForLobbyPage={isInLobbyOrConnecting}
+      callControls={props.callControls}
+    />
+  );
+
+  /* @conditional-compile-remove-from(stable) */
+  if (isMobile) {
+    return (
+      <Stack className={allHeightAndWidthStyle}>
+        {callAdapter && chatProps.adapter && hasJoinedCall && (
```

Comment:
what happens if hasJoinedCall is false? It seems to be required for all of the components we are going to return. If we just need it in all cases then perhaps  

If(!hasJoinedCall) then just return?